JASP-Desktop/components/JASP/Controls/TableView.qml

```diff
@@ -196,20 +200,18 @@ JASPControl
 				leftTopCornerItem: Rectangle
 				{
 					color: jaspTheme.analysisBackgroundColor
+					width: cornerText.width
```

Comment:
This is overwritten by `DataSetView::createleftTopCorner()` and because you turned eliding off this might make things *very* ugly.